frontend/src/common/ModelImageList.tsx

```diff
@@ -0,0 +1,59 @@
+import Autocomplete from '@mui/material/Autocomplete'
+import TextField from '@mui/material/TextField'
+import { useGetModelImages } from 'actions/model'
+import { Dispatch, SetStateAction, SyntheticEvent, useEffect, useState } from 'react'
+import { ModelInterface } from 'types/v2/types'
+
+import { FlattenedModelImage } from '../../types/interfaces'
+import Loading from '../common/Loading'
+import MessageAlert from '../MessageAlert'
+
+interface ModelImageListProps {
+  model: ModelInterface
+  value: FlattenedModelImage[]
+  setImages: Dispatch<SetStateAction<FlattenedModelImage[]>>
+}
+
+export default function ModelImageList({ model, value, setImages }: ModelImageListProps) {
+  const { modelImages, isModelImagesLoading, isModelImagesError } = useGetModelImages(model.id)
+  const [flattenedImageList, setFlattenedImageList] = useState<FlattenedModelImage[]>([])
+
+  useEffect(() => {
+    if (modelImages) {
```

Comment:
I think this will always be an array, even if an empty one. So this check will always pass